requirements/edx/base.txt

```diff
@@ -771,7 +778,7 @@ pymongo==3.10.1
     #   mongoengine
 pynliner==0.8.0
     # via -r requirements/edx/base.in
-pyparsing==3.0.1
+pyparsing==2.4.7
```

Comment:
`packaging==21.2` requires `pyparsing<3.0` so the version is downgrading. It'll be fixed in the next version of `packaging` and will be updated back.